src/styles/scss/abstracts/_mixins.scss

```diff
@@ -12,5 +12,81 @@
   font-style: $font-style;
   font-size: $font-size;
   line-height: $line-height;
-  color: $color;
+  color: $custom-black;
+}
+
+@mixin custom-text {
```

Comment:
* 1 mixin for font size
* on html we have font-family

Reply:
done